app/components/candidate_interface/offer_review_component.rb

```diff
@@ -27,14 +27,14 @@ def provider_row
     def course_row
       {
         key: 'Course',
-        value: @course_choice.offered_course.name_and_code,
+        value: course_row_value,
       }
     end
 
     def location_row
       {
         key: 'Location',
-        value: @course_choice.offered_option.site.name,
+        value: @course_choice.offered_option.site.name_and_address,
```

Comment:
We didn't deliberately change this - I think it was an existing difference between the prototype and production. Possibly it's still worth doing though?

Reply:
I'll revert it for now since it was an accidental change, but happy to revert it if you'd like me too.